features/full_tests/native_feature_flags.feature

```diff
@@ -1,11 +1,10 @@
 Feature: Synchronizing feature flags to the native layer
 
   Scenario: Feature flags are synchronized to the native layer
```

Comment:
It'd be good to get an understanding of why this test passed previously and make any changes based on that. As it stands it looks like the test should have been failing previously.

Reply:
As written, I don't see how it could have ever ran successfully - the app would not have relaunched.